.github/CONTRIBUTING.md

```diff
@@ -1,134 +1,94 @@
-## Pull Requests
+## Terms
 
--   Every PR must be tied to an issue. Create an issue if none exist for the PR.
--   PRs should only be scoped to a single issue.
--   PR commits should contain [meaningful commit messages].
--   Please fill out the entirety of the PR template, including the PR checklist.
--   If your PR adds a feature with a public API, please add or update documentation in the README for the relevant package.
+**Core Team**
+: The Magento core engineering/product team working on the PWA Studio project.
 
-**Note:**
-_As a developer, please write developer-facing documentation in README or other Markdown files in the relevant package files. Do not add documentation directly to the `pwa-devdocs` folder. That folder is maintained by the Technical Writing team, who will revise and proofread your documentation and migrate it to the devdocs site._
+**Community Maintainer**
+: A community point of contact approved by the Core Team to help with project administration.
 
-## Contribution process
+Current list of maintainers:
 
-Magento maintains a public roadmap for this and other repositories in each project's issue board.
+-   [Jordan Eisenburger](https://github.com/Jordaneisenburger)
 
-Any and all community participation in this backlog is encouraged and appreciated.
-Even foundational infrastructure stories are available for a generous developer to take on.
+**Community Advisory Board**
+: Consists of Community Maintainer(s) and Core Team members who help drive prioritization and scoping for community driven **help wanted** features and enhancements.
 
-To get started, look for issues tagged with the **[help wanted]** labels.
-These issues are ready for community ownership.
+**Directly Responsible Individual**
+: The assigned developer for a specific issue.
+This person is responsible for ensuring the issue is completed in a reasonable amount of time and to a certain standard.
 
-### Claiming an issue on the roadmap
+## Ways to contribute
 
-If you are interested in taking ownership of a roadmap feature or issue, we ask that you go through the following process.
-This helps us organize and forecast the progress of our projects.
+-   [Identify and create issues](#identify-and-create-issues)
+-   [Select a groomed issue](#select-a-groomed-issue)
+-   [Help answer community questions](#help-answer-community-questions)
 
-#### Step 1: Add an issue comment
+### Identify and create issues
 
-Add a comment on an issue expressing your interest in taking ownership of it.
-Make sure your GitHub profile includes an email address so we can contact you privately.
+If you encounter an issue while using PWA Studio or you have a suggestion for a new feature, let us know by [creating an issue][].
+Provide as much detail as you can to help us reproduce or analyze the issue before prioritizing it.
 
-#### Step 2: Meet with a maintainer
+When you create an issue, add the **help wanted** label to bring it to the attention of the Community Advisory Board.
```

Comment:
Community members will not have the ability to add this label, so I would say to ping us in Slack, or just say we will go through newly created issues and add the label to issues we deem suitable for the community.